src/views/AboutView.vue

```diff
@@ -52,6 +52,7 @@ const tagline = computed(() => TAGLINES[Math.floor(Math.random() * TAGLINES.leng
         Forrest Cahoon - <i>developer</i> <br />
         Videco - <i>developer</i> <br />
         Inthar - <i>developer</i> <br />
+        Marc Sabat - <i>composer / developer</i> <br />
```

Comment:
Technically Scale Workshop does not ship with a soundtrack, so maybe *notation advisor* or *hardware specialist* instead of *composer*.

Reply:
fixed